.github/actions/upload-flaky-tests/action.yml

```diff
@@ -0,0 +1,44 @@
+name: Report flaky test
+description: Finds flaky tests and creates issues
+
+runs:
+  using: composite
+  steps:
+    - id: flaky-tests
+      name: Find flaky tests
+      if: github.repository == 'stianst/keycloak'
+      shell: bash
+      run: |
+        REPO="${{ github.repository }}"
+        ISSUES_LINK="https://github.com/${{ github.repository }}/issues/"
+        PR="${{ github.event.pull_request.number }}"
+        RUN="https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+        for dir in $(find -type d -name surefire-reports); do
+          for i in $(ls $dir/*.txt); do
+            if ( grep '<<< FAILURE!' $i &>/dev/null ); then
+              TEST="$(echo $i | sed 's/.txt//g' | sed 's/.*\.//g')"
+              LOG="$(cat $i)"
+              
+              TITLE="Flaky test: $TEST"
+              BODY="$RUN"$'\n''```'$'\n'"$LOG"$'\n''```'
+              ISSUE="$(gh issue list --search "$TITLE in:title" --json number --jq .[].number)"
```

Comment:
The issue should be an open issue, not just any issue with that title.

Reply:
`gh issue list` only returns open issues. I've tested this and it does indeed create a new issue if there's a closed issue with the same title.

Reply:
Oh, didn't know that. Consider it resolved!